select_interface.py

```diff
@@ -0,0 +1,20 @@
+import inquirer
+import netifaces
+import os
+questions = [
+  inquirer.List('Network Interface',
+                message="Select Interfaces ?",
+                choices= netifaces.interfaces(),
+            ),
+]
+answers = inquirer.prompt(questions)
```

Comment:
We cannot rely on prompting for user input. If SRE were to deploy 1000 server, s/he will not enter the answer one by one.

Reply:
Agree. Can we use a configuration file or a command line parameter to pass in the desired interface name?  If that doesn't exist, then return the  default interface name "eth0".